src/main/scala/com/gu/fulfilmentLookup/Lambda.scala

```diff
@@ -6,27 +6,37 @@ import java.time.format.DateTimeFormatter
 import play.api.libs.json.{ JsError, JsSuccess, JsValue, Json }
 import com.amazonaws.services.lambda.runtime.Context
 import com.gu.fulfilmentLookup.ResponseWriters._
-import scala.util.{ Failure, Success }
+import scala.util.{ Failure, Success, Try }
 import scalaz.{ -\/, \/- }
 
 trait FulfilmentLookupLambda extends Logging {
 
-  def s3Client: CsvClient
+  def csvClient: CsvClient
   def caseService: CaseService
```

Comment:
not part of this PR, but caseService you only use raiseCase, why not pass in the function instead of the whole service? or rename the class to RaiseCase if you want a simpler type.  If we wanted a ListCases it probably wouldn't make sense that any code that needs to Raise cases would also get that code as a dependency.